components/CompositionCollectionView/src/IsExternalInit.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We should just use [PolySharp](https://github.com/Sergio0694/PolySharp) reference for these. Were you using records somewhere?

Reply:
PolySharp is included by default on the Uwp head. For component libraries, they need to be included on the project itself, see examples [here](https://github.com/search?q=repo%3ACommunityToolkit%2FWindows%20PolySharp&type=code).